packages/cli/templates/uiPlugin/src/public/manifest.json

```diff
@@ -2,11 +2,11 @@
     "urn": "vmware:vcloud:plugin:<%- name %>",
     "name": "<%- nameCamelCase %> plugin",
     "containerVersion": "10.2.0",
-    "version": "<%- components.ui.version %>",
+    "version": "<%- components.uiPlugin.version %>",
     "scope": ["service-provider", "tenant"],
     "permissions": [],
     "description": "<%- nameCamelCase %> Plugin",
```

Comment:
This should be description.

Reply:
We are not asking for description, so we should probably change it to "Description for <%- nameCamelCase %> Plugin" and the author can change it afterwards. Will update that in the next pull request